client/jetpack-connect/remote-credentials.js

```diff
@@ -219,12 +227,12 @@ export class OrgCredentialsForm extends Component {
 					{ this.isInvalidUsername() && (
 						<FormInputValidation
 							isError
-							text={ translate( 'Username does not exist. Please try again.' ) }
+							text={ translate( 'Username or email does not exist. Please try again.' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.